Getting-Started/Backoffice/Property-Editors/Built-in-Property-Editors/Grid-Layout/Grid-Editors.md

```diff
@@ -86,3 +87,18 @@ The `config.markup` is the string rendered server side in your template. `#value
 
 In this sample `config.size` will resize the the image according to `height` and `width`. The above example will result in a rendered image that is 200x200 pixels no matter the size of the uploaded image. If the ratio of the size differs from the uploaded image it is possible to set a focal point that determines how the image should be cropped.
 ![Resizing](images/grid-resizing.png)
+
+### package.manifest
+
+As an alternative to `/config/grid.editors.config.js`, you can also define grid editors through a `package.manifest` file. If you create a new file at the `/App_Plugins/{YourPackageName}/package.manifest`, Umbraco will automatically pick it up. While the root JSON element of `/config/grid.editors.config.js` is an array of grdi editors, `package.manifest` files start with a JSON object with a number of different properties - one of them being `gridEditors`:

```

Comment:
Shouldn't say as an alternative. You should never use grid.editors.config as stated earlier in the article the recommended approach is to use package.manifest. The code example here is also a repetition of the one earlier so maybe they could be combined?